src/styles/index.scss

```
@@ -1,3 +1,136 @@
-body {
+$dark-color: #060b35;
+$light-color: #616070;
+$button-color: #00acdc;
+$width: 166px;
+$button-height: 40px;
+
+.page {
   margin: 0;
+  padding: 0;
+
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
```

```suggestion
  font-family: "Roboto", sans-serif;
  font-style: normal;
  font-weight: 400;
  font-size: 10px;
```
this styles better to set in html selector